containers/autoReply/AutoReplyTemplate.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { c } from 'ttag';
+import PropTypes from 'prop-types';
+import { Button } from 'react-components';
+import moment from 'moment';
+import {
+    getDaysOfMonthOptions,
+    getDurationOptions,
+    AutoReplyDuration,
+    DAY_MILLISECONDS,
+    getTimeZoneOptions
+} from './utils';
+import InfoLine from './InfoLine';
+
+const AutoReplyTemplate = ({ autoresponder, onEdit }) => {
+    const durationLabel = getDurationOptions().find(({ value }) => value === autoresponder.Repeat).text;
+    const timezone = getTimeZoneOptions().find(({ value }) => value === autoresponder.Zone).text;
+
+    const formatTime = (time) => {
+        const hours = moment.utc(time).format('LT');
+
+        if (autoresponder.Repeat === AutoReplyDuration.DAILY) {
+            const firstDayOfWeek = moment.localeData().firstDayOfWeek();
+            const orderedDaysSelected = [
+                ...autoresponder.DaysSelected.filter((day) => day >= firstDayOfWeek).sort((a, b) => a - b),
+                ...autoresponder.DaysSelected.filter((day) => day < firstDayOfWeek).sort((a, b) => a - b)
+            ];
+            const weekdays = orderedDaysSelected.map((day) => moment.weekdaysShort(day)).join(', ');
+            return autoresponder.DaysSelected.length < 7
+                ? c('Duration').t`Every ${weekdays} @ ${hours}`
+                : c('Duration').t`Every day @ ${hours}`;
+        } else if (autoresponder.Repeat === AutoReplyDuration.FIXED) {
+            const date = moment(time).format('LL');
+            return `${date} @ ${hours}`;
+        } else if (autoresponder.Repeat === AutoReplyDuration.WEEKLY) {
+            const dayOfWeek = moment.weekdays(Math.floor(time / DAY_MILLISECONDS));
+            return c('Duration').t`Every ${dayOfWeek} @ ${hours}`;
+        } else if (autoresponder.Repeat === AutoReplyDuration.MONTHLY) {
+            const dayOfMonth = getDaysOfMonthOptions().find(
+                ({ value }) => value === Math.floor(time / DAY_MILLISECONDS)
+            ).text;
+            return c('Duration').t`Every ${dayOfMonth} @ ${hours}`;
+        } else if (autoresponder.Repeat === AutoReplyDuration.PERMANENT) {
+            return c('Duration').t`n/a`;
+        }
+    };
+
+    return (
+        <div className="bordered-container p2 mw650p">
+            <table>
```

Comment:
Why not using a structure with `Row`, `Label` and `Field`?

Reply:
From the design it seemed that the all of the labels must have the same width and take it from the widest one. That's basically a table. If I use Rows, labels would have different sizes or I would need to set a fixed size for them. I've hard opinion on this, just it seemed to be simpler this way.